operators/src/processing/reprojection.rs

```diff
@@ -1092,10 +1054,17 @@ mod tests {
             .await
             .unwrap();
 
-        qs.map(Result::unwrap)
+        let tiles = qs
+            .map(Result::unwrap)
             .collect::<Vec<RasterTile2D<u8>>>()
             .await;
 
+        // the test must generate 8x4 tiles
+        dbg!(tiles.len(), 32);
+
+        // none of the tiles should be empty
+        assert!(tiles.iter().all(|t| !t.is_empty()));
```

Comment:
should we check the content for correctness also?

Reply:
i guess we should do that with an other test